model/trace/rntbd.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: rntbd
+    type: span
+    span_kind: client
+    brief: >
+      Network-level attributes for Cosmos DB. It is Microsoft proprietry protocol for Cosmos DB.
+    attributes:
```

Comment:
do you want to it include any  other attributes like `server.address`? 
See my comment above for url - standard attributes would create better user experience (visualizations don't need to know about rntbd, dashboards, alerts and custom queries would be much easier with standard attributes.